cirq-google/cirq_google/api/v2/run_context.proto

```diff
@@ -10,6 +12,10 @@ option java_multiple_files = true;
 message RunContext {
   // The parameters for operations in a program.
   repeated ParameterSweep parameter_sweeps = 1;
+
+  // optional override of select device parameters before program
```

Comment:
good catch. yes the proto definition allows such overlap. however the override will have no effect since it is applied before the circuit sweep is executed.
I have put in proto field comment to document such case.